src/test/java/org/example/Homework11Test.java

```diff
@@ -0,0 +1,37 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+
+public class Homework11Test {
+
+    private final Homework11 homework11 = new Homework11();
+
+    @Test
+    void testMaxDuplicates() {
+        List<Integer> numbers = List.of(1, 2, 2, 1, 1);
+        int actualResult = homework11.maxDuplicate(numbers);
+        assertEquals(2, actualResult);
+    }
```

Comment:
The point of tests is to try out different scenarios. Add tests for the case, when the list is empty, when there are no duplicates